test/framed.test.js

```diff
@@ -0,0 +1,40 @@
+import { expect } from '@open-wc/testing';
+import frauJwt from '../src/framed.js';
+import REQUEST_KEY from '../src/request-key.js';
+import sinon from 'sinon';
+
+const TOKEN_VALUE = 'token-val!';
+
+describe('framed', () => {
+
+	let messageCallback, sandbox;
+	beforeEach(() => {
+		sandbox = sinon.createSandbox();
```

Comment:
I'm basically stubbing out `ifrau` here... it knows a little bit too much about how ifrau works, but I couldn't think of a good alternative.